lib/shopify-cli/identity_auth.rb

```diff
@@ -237,13 +237,15 @@ def challange_params
     def auth_url
       if Environment.use_local_partners_instance?
         "https://identity.myshopify.io/oauth"
+      elsif Environment.use_spin_partners_instance?
+        "https://identity.#{Environment.spin_url}/oauth"
       else
         "https://accounts.shopify.com/oauth"
       end
```

Comment:
@mkcny and I talked and this might be something we need to support with the installed version of the CLI. We can simplify the logic as we remove the stubbed and local configurations for scripts and solely rely on spin instances.